eng/pipelines/templates/steps/install-dotnet.yml

```diff
@@ -16,7 +16,15 @@ steps:
     retryCountOnTaskFailure: 3
     inputs:
       # We must install sdk, not just runtime, as it is required by some of our tools, like test-proxy.
-      # For additional context, see PR: https://github.com/Azure/azure-sdk-tools/pull/5405
+      # Specifically, test-proxy requires asp.net core runtime, which is installed only when sdk option
```

Comment:
Ahhhhhh so it's not that there was anything with the runtime we were using, it just didn't have the `asp.net core` runtime in the standard. 👍